mmdet/structures/track_data_sample.py

```diff
@@ -0,0 +1,272 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import List, Optional, Sequence
+
+import numpy as np
+import torch
+from mmengine.structures import BaseDataElement
+
+from .det_data_sample import DetDataSample
+
+
+class TrackDataSample(BaseDataElement):
```

Comment:
This datasample is designed only for tracking or for all video tasks?

Reply:
It's only used in tracking task